File: 04_quicksort/python/01_loop_sum.py
'''
@Description: In User Settings Edit
@Author: your name
@Date: 2019-08-21 20:16:55
@LastEditTime: 2019-08-21 20:19:42
@LastEditors: Please set LastEditors
'''
def sum(arr):
    total = 0
    for i in range(len(arr)):
        total += arr[i]
    return total

arr = [1,2,3]
print(sum(arr))

File: 04_quicksort/go/01_loop_sum.go
/*
 * @Description: In User Settings Edit
 * @Author: your name
 * @Date: 2019-08-21 20:20:56
 * @LastEditTime: 2019-08-22 10:10:48
 * @LastEditors: Please set LastEditors
 */
package main

import "fmt"

func sum(arr []int) int {
	total := 0
	for i := 0; i < len(arr); i++ {
		total += arr[i]
	}
	return total
}

var arr = [3]int{1, 2, 3}

func main() {
	fmt.Println(sum([]int{1, 2, 3, 4}))
}
